addons/stock_account/views/product_views.xml

```diff
@@ -40,6 +40,7 @@
             <field name="name">product.template.stock.property.form.inherit</field>
             <field name="model">product.template</field>
             <field name="inherit_id" ref="product.product_template_form_view"/>
+            <field name="groups_id" eval="[(4, ref('stock.group_stock_manager'))]"/>
```

Comment:
There's an issue here as `mrp.group_mrp_manager` doesnt have `stock.group_stock_manager` as `implied_ids`: https://github.com/odoo/odoo/blob/96406dc2a9e06595961f6d4bec6b66f27b94e2ff/addons/mrp/security/mrp_security.xml#L18

This causes that users who are MRP managers but only Stock users are going to get an inheritance error when trying to get to any product view.